File: clients/typescript-angular/API/Client/Settlement.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Settlement {

        timestamp: date;

        symbol: string;

        settlementType?: string;

        settledPrice?: number;

        bankrupt?: number;

        taxBase?: number;

        taxRate?: number;
    }

}

File: servers/scalatra/src/main/scala/com/wordnik/client/model/Settlement.scala
package com.wordnik.client.model

import java.util.Date
import java.math.BigDecimal


case class Settlement (
  timestamp: Date,
  symbol: String,
  settlementType: String,
  settledPrice: Double,
  bankrupt: BigDecimal,
  taxBase: BigDecimal,
  taxRate: Double
)
